noir-projects/noir-protocol-circuits/crates/types/src/shared_mutable/scheduled_delay_change.nr

```diff
@@ -15,10 +14,10 @@ pub struct ScheduledDelayChange<let INITIAL_DELAY: u32> {
     // they default to Option::none(), which we detect and replace with INITIAL_DELAY. The end result is that a
     // ScheduledDelayChange that has not been initialized has a delay equal to INITIAL_DELAY, which is the desired
     // effect. Once initialized, the Option will never be none again.
-    pre: Option<u32>,
-    post: Option<u32>,
+    pub(crate) pre: Option<u32>,
+    pub(crate) post: Option<u32>,
     // Block at which `post` value is used instead of `pre`
-    block_of_change: u32,
+    pub(crate) block_of_change: u32,
```

Comment:
Had to do this change because now we need direct access to it when packing in `shared_mutable_values.nr`. Not great.